src/components/ResourceListPrefix.svelte

```diff
@@ -0,0 +1,26 @@
+<svelte:options immutable={true} />
+
+<script lang="ts">
+  import type { ResourceType } from '../types/simulation';
+  import type { TimelineItemType } from '../types/timeline';
+
+  export let item: TimelineItemType | undefined = undefined;
+
+  let prefix: string = '';
+
+  $: units = (item as ResourceType)?.schema.metadata?.unit?.value;
```

Comment:
do you mind declaring the types for `units` and `type` before the reactive statements?